atom/renderer/lib/init.js

```diff
@@ -36,6 +36,15 @@ electron.ipcRenderer.on('ELECTRON_INTERNAL_RENDERER_WEB_FRAME_METHOD', (event, m
   electron.webFrame[method].apply(electron.webFrame, args);
 });
 
+electron.ipcRenderer.on('ELECTRON_INTERNAL_RENDERER_ASYNC_WEB_FRAME_METHOD', (event, method, args) => {
+  let requestId = args.shift();
```

Comment:
If fist element of `args` is always `requestId`, we should add it in parameters: `(event, method, requestId, args)`.